bundles/net.tourbook/plugin.properties

```diff
@@ -200,7 +200,7 @@ View_Name_TourMap25_Tooltip                  = Show tour in 2.5D map
 View_Name_TourMap3                           = 3D Tour Map
 View_Name_TourMap3_Mnemonic                  = 3
 View_Name_TourMap3_Tooltip                   = Show tour in 3D map
-View_Name_TourMarkerAll                      = Tour Marker
+View_Name_TourMarkerAll                      = All Tour Markers
```

Comment:
Translators cannot recognize this change, you should delete `View_Name_TourMarkerAll` and use the new text key `View_Name_TourMarkerAll_2`

Reply:
Ok, in this case, it might even be better to give a complete new better name as the previous one was not accurate.
Renamed to `View_Name_AllTourMarkers`